Added how to get number of builds between two dates (#2130)
buildkite/docs: pages/apis/graphql/graphql_cookbook.md

  }
}
```

## Test Analytics

> ### Get number of builds between 2 dates 

Reviewer: Style nitpicks: 

- Write out number for 0-9.
- Add a blank line between headings and paragraphs.

```suggestion
### Get number of builds between two dates
 
```

> There is a correlation between number of builds and job minutes used. Though this query won't get you how many job minutes have been used, but it will give an idea of whether or not there is an increase in the number of builds, which will cause more job minutes to be used.

Reviewer: What's the main task that people are trying to do here? Is it to look for usage trends and this is the recommended way? If so, maybe the heading should be something like, _Check usage in a period_ and the text:

```suggestion
This query helps you understand how many job minutes you've used by looking at the number of builds. While not equivalent, there's a correlation between the number of builds and job minutes. So, looking at the number of builds in different periods gives you an idea of how the job minutes would compare in those periods.
```

Also, is this connected to TA? I thought job minutes were a Pipelines thing? And the equivalent in TA is test executions?

Author: You are correct! For some reason I was thinking TA, but it is under the pipeline subheading in the rest of the documentation. I blame the Friday evening slump.

I'll correct this!

```graphql

> query PipelineBuildCountForBranchQuery {

Reviewer: I think this is a copy/paste hangover, so I think it should be updated. Perhaps:

```suggestion
query PipelineBuildCountForPeriod {
```
